### ISSUE TITLE
It will be great to have a seporate list of Obsolete methods

### ISSUE THREAD

**Issue by SergeyKanzhelev** [DotNetAnalyzers/PublicApiAnalyzer#9]

**Comment by sharwell** [comment]
I agree. It would be good to also post this to [dotnet/roslyn-analyzers](https://github.com/dotnet/roslyn-analyzers), as this repository is just a back-port of one of the analyzers in that repository to work with older versions of Roslyn.

**Comment by SergeyKanzhelev** [comment]
Which NuGet package should I use then?
